src/script.js

```diff
@@ -46,11 +43,12 @@ const createGame = async () => {
   const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/', {
     method: 'POST',
     headers: { 'Content-Type': 'application/json' },
-    body: JSON.stringify({ name: 'YOUR_GAME_NAME' }),
+    body: JSON.stringify({ name: 'RACING HORSE' }),
   });
   const data = await response.json();
-  // eslint-disable-next-line prefer-destructuring
-  GAME_ID = data.result.split('games/')[1].split('/')[0];
+  const [, GAME_ID] = data.result.split('games/');
+  // eslint-disable-next-line no-console
+  console.log(GAME_ID);
```

Comment:
Just as the previous reviewer requested, kindly take away the commands you have added to disable `eslint` in your codebase and fix the linter errors on yourself😉 as we add linters in our files to enable us to write better code👍

also, I suggest you take away the `console.log` you  used in debugging as it makes your code looks messy and less professional👍